challenges/challenge_study_schedule.py

```diff
@@ -1,8 +1,14 @@
 def study_schedule(start_time, end_time, target_time):
-    # Faça o código aqui.
+    count = 0
+    if target_time < 0 or target_time > 23:
+        return count
+    for idx, values in enumerate(start_time):
+        if (target_time >= start_time[idx] and target_time <= end_time[idx]):
```

Comment:
Vi o John usando isso, acho que no pai ta on funciona
```suggestion
        if (start_time[idx] <= target_time <= end_time[idx]):
```

Reply:
Nossa, farei isso com certeza